_monitoring-plugins/ad/index.md

```diff
@@ -25,21 +25,37 @@ To first test with sample streaming data, you can try out one of the preconfigur
-Setting the window delay to 1 minute shifts the interval window to 1:49 - 1:59, so the detector accounts for all 10 minutes of the detector interval time.
-1. Choose **Next**.
+1. Add in the detector details.
+   - Enter a name and brief description. Make sure the name is unique and descriptive enough to help you to identify the purpose of the detector.
+1. Specify the data source.   
+   - For **Data source**, choose the index you want to use as the data source. You can optionally use index patterns to choose multiple indices.
+   - (Optional) For **Data filter**, filter the index you chose as the data source. From the **Data filter** menu, choose **Add data filter**, and then design your filter query by selecting **Field**, **Operator**, and **Value**, or choose **Use query DSL** and add your own JSON filter query.
+1. Specify a timestamp.    
+   - Select the **Timestamp field** in your index.
+1. Define operation settings.
+   - For **Operation settings**, define the **Detector interval**, which is the time interval at which the detector collects data.
+      - The detector aggregates the data in this interval, then feeds the aggregated result into the anomaly detection model.
+      The shorter you set this interval, the fewer data points the detector aggregates.
+      The anomaly detection model uses a shingling process, a technique that uses consecutive data points to create a sample for the model. This process needs a certain number of aggregated data points from contiguous intervals.
+
+      - We recommend setting the detector interval based on your actual data. If it's too long it might delay the results, and if it's too short it might miss some data. It also won't have a sufficient number of consecutive data points for the shingle process.
+
+   - (Optional) To add extra processing time for data collection, specify a **Window delay** value.
+      - This value tells the detector that the data is not ingested into OpenSearch in real time but with a certain delay. Set the window delay to shift the detector interval to account for this delay.
+      - For example, say the detector interval is 10 minutes and data is ingested into your cluster with a general delay of 1 minute. Assume the detector runs at 2:00. The detector attempts to get the last 10 minutes of data from 1:50 to 2:00, but because of the 1-minute delay, it only gets 9 minutes of data and misses the data from 1:59 to 2:00. Setting the window delay to 1 minute shifts the interval window to 1:49 - 1:59, so the detector accounts for all 10 minutes of the detector interval time.
+1. Specify custom result index.
+   - If you want to store the anomaly detection results in your own index, choose **Enable custom result index** and specify the custom index to store the result.
+      - If the custom index you specify doesn’t already exist, the anomaly detection plugin creates this index when you create the detector and start your real-time or historical analysis.
```

Comment:
Also add these descriptions?

The prefix “opensearch-ad-plugin-result-” will be added to the index name that you input. For example, if you input “abc” as the result index name, the final index name will be “opensearch-ad-plugin-result-abc.”

You can use dash “-” to separate the namespace to manage custom result index permissions easily. For example, if you use “opensearch-ad-plugin-result-financial-us-group1” as result index, you can create a permission role based on pattern 
"opensearch-ad-plugin-result-financial-us-*" for “financial” department at a granular level for the “us” area.